sympy/functions/elementary/hyperbolic.py

```diff
@@ -926,6 +927,19 @@ def eval(cls, arg):
                 if _coeff_isneg(arg):
                     return -cls(-arg)
 
+        if isinstance(arg, sinh):
+            x = arg.args[0]
```

Comment:
I would first check that `x` is a number and only then go on. Otherwise, I think that automatic simplification should be avoided.